libmpcdec/mpc_bits_reader.c

```diff
@@ -174,8 +174,5 @@ int mpc_bits_get_block(mpc_bits_reader * r, mpc_block * p_block)
 	if (p_block->size >= size) // check if the block size doesn't conflict with the header size
 		p_block->size -= size;
 
-	return size;
+	return (int)size;
```

Comment:
Is it possible to drop the cast by changing this return type?

Reply:
This is, as far as we can tell, part of the public API, so the only way to do that is an ABI break - that's why we've avoided it.